tool/jt.rb

```diff
@@ -1354,14 +1352,19 @@ def mspec(*args)
     run_mspec env_vars, command, *options, *prefixed_ruby_args, *args
   end
 
+  def gem_test_pack?
+    return true if ci?
+    Dir.exist?(File.expand_path('truffleruby-gem-test-pack', TRUFFLERUBY_DIR))
+  end
```

Comment:
@chrisseaton @kirs (#1736) FYI, I changed this so it's automatically detected whether the gem test pack is available, without needing to pass extra flags.

Reply:
Thanks, that sounds like a good idea!